charmdir.go

```diff
@@ -331,20 +334,33 @@ func checkFileType(path string, mode os.FileMode) error {
 }
 
 // MaybeGenerateVersionString generates charm version string.
-func (dir *CharmDir) MaybeGenerateVersionString() (string, error) {
+func (dir *CharmDir) MaybeGenerateVersionString() (string, string, error) {
```

Comment:
Maybe worth a comment what a 2nd returned string is?

Reply:
ok